_api-reference/index-apis/stats.md

```diff
@@ -0,0 +1,647 @@
+
+### Metrics
+
+The following table lists all available metric groups.
+
+Metric | Description
+:--- |:----
+`_all` | Return all statistics. 
+`completion` | Completion suggester statistics. 
+`docs` | Returns the number of documents and the number of deleted docs that have not yet been merged. Index refresh operations can affect this statistic. 
+`fielddata` | Field data statistics. 
+`flush` | Flush statistics. 
+`get` | Get statistics, including missing stats. 
+`indexing` | Indexing statistics. 
+`merge` | Merge statistics. 
+`query_cache` | Query cache statistics. 
+`refresh` | Refresh statistics. 
+`request_cache` | Shard request cache statistics. 
+`search` | Search statistics, including suggest operation statistics. Search operations can be associated with one or more groups. You can include statistics for custom groups by providing a `groups` parameter, which accepts a comma-separated list of group names. To return statistics for all groups, use `_all`. 
+`segments` | Statistics about memory use of all open segments. If the `include_segment_file_sizes parameter` is `true`, this metric includes the aggregated disk usage of each Lucene index file.
+`store` | Size of the index in byte units. 
+`translog` | Translog statistics. 
+`warmer` | Warmer statistics. 
+
+## Query parameters
+
+The following table lists the available query parameters. All query parameters are optional.
+
+Parameter | Data type | Description 
+:--- | :--- | :--- 
+`expand_wildcards` | String | Specifies the type of indexes to which wildcard expressions can expand. Supports comma-separated values. Valid values are: <br> - `all`: Expand to all open and closed indexes, including hidden indexes <br> `open`: Expand to open indexes <br> - `closed`: Expand to closed indexes <br> - `hidden`: Include hidden indexes when expanding. Must be combined with `open`, `closed`, or both <br> - `none`: Do not accept wildcard expressions <br> Default is `open`.
+`fields` | String | A comma-separated list or a wildcard expression specifying fields to include in the statistics. Specifies the default field list if neither `completion_fields` nor `fielddata_fields` are provided.
+`completion_fields` | String | A comma-separated list or wildcard expression specifying fields to include in field-level `completion` statistics.
+`fielddata_fields` | String | A comma-separated list or wildcard expression specifying fields to include in field-level `fielddata` statistics.
+`forbid_closed_indices` | Boolean | Specifies not to collect statistics for closed indexes. Default is `true`.
+`groups` | String | A comma-separated list of search groups to include in the `search` statistics.
+`level` | String | Specifies the level to aggregate statistics. Valid values are: <br> - `cluster`: Cluster-level statistics, `indices`: Index-level statistics <br> - `shards`: Shard-level statistics. <br> Default is `indices`.
```

Comment:
"the level used to aggregate"? Also, should `indices` be its own bullet point? And they should all end with a period.